.github/workflows/stage-build.yml

```
@@ -229,6 +229,8 @@ jobs:
 
           DEPLOYER_ELASTICSEARCH_URL: ${{ secrets.DEPLOYER_STAGE_ELASTICSEARCH_URL }}
 
+          LAMBDA_PREFIX: stage-
```

Nit: Shouldn't we prefix this env - and move it next to the other DEPLOYER envs?

```suggestion
          DEPLOYER_LAMBDA_PREFIX: stage-
```